socorro/external/postgresql/setupdb_app.py

```diff
@@ -162,15 +162,6 @@ class SocorroDBApp(App):
         exclude_from_dump_conf=True
     )
 
-    @staticmethod
-    def get_application_defaults():
-        """since this app is more of an interactive app than the others, the
-        logging of config information is rather disruptive.  Override the
-        default logging level to one that is less annoying."""
-        return {
-            'logging.stderr_error_logging_level': 40  # only ERROR or worse
-        }
-
```

Comment:
I removed this because it makes it impossible to figure out what `setupdb_app` did. The output it produces it ornery, but it happens once when building a dev environment and it's more important to know what it did. If someone doesn't like it, they can adjust the logging level via the command line.